### PR TITLE
Update Language Settings

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update Language Settings File

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -95,10 +95,10 @@ function Get-java-PackageInfoFromPackageFile ($pkg, $workingDirectory) {
 function Publish-java-GithubIODocs ()
 {
   $PublishedDocs = Get-ChildItem "$DocLocation" | Where-Object -FilterScript {$_.Name.EndsWith("-javadoc.jar")}
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:97]: Need to also update PackageInfoFromPackageFile  to add ReleaseTag

Comment (chidozieononiwu) [reply eng/scripts/Language-Settings.ps1:97]: I'll do that when am focusing on that part of the code. There is the chance that it changes before I get to work on it. Right now I want to get the copy to blobs part done

**Review by weshaggard — APPROVED** [review]

**Pull request merged**